src/public/index.html

```diff
@@ -9,37 +9,47 @@
   </head>
 
 
-    <body>
-      <header>
-        <h1>Something Awesome!</h1>
-      </header>
-      <nav>
-        <ul>
-          <li><a href="/about/">About</a></li>
-          <li> <a href="/contact/">Contact</a></li>
-        </ul>
-      </nav>
+  <body>
+    <header>
+      <div class="title">
+      <h1 class="plaintext-color">Something Awesome! <br/> </h1>
+    </div>
```

Comment:
Indentation is important. Try and keep the opening and closing tags at the same level of indentation, otherwise the markup becomes hard to read.